### DISCUSSION TITLE
Double exposure capability oddity

### DISCUSSION THREAD

**Discussion by futrtrubl** [sn4k3/UVtools#366] [Q&A] [answered]
Updating Lychee somewhere, UVtools now says my Elegoo Mars 2 is no longer capable of Double Exposure. Old .ctb files imported allow double exposure, while new ones give the pop up saying "The current printer and/or file format is not compatible with this tool. In the zip are examples of both.
[files.zip](https://github.com/sn4k3/UVtools/files/7689239/files.zip)
"

**Comment by sn4k3** [comment, marked as answer]
Working is CTBv3
NotWorking is CTBv2

You need CTBv3 at least for that tool to work. Lychee last version allow to choose CTB version, choose v3 or v4

**Reply by futrtrubl** [reply]
Thank you very much! I never changed that setting, never knew that setting even existed... Works now that I changed it to v4.
